benchmark/app.py

```diff
@@ -0,0 +1,94 @@
+import os
+import time
+import uuid
+from math import ceil
+
+import click
+from pandas import DataFrame
+
+from jina.flow import Flow
+from jina.drivers import BaseDriver
+
+from post_process import clean_dataframe, evaluate_times, get_summary
+from utils import bytes_generator, generate_filename, get_list_of_num_docs
+from summary import html_table, plot_num_docs_vs_time
+
+FILE_DIR = os.environ.get('FILE_DIR', './.data')
+os.makedirs(FILE_DIR, exist_ok=True)
+
+START_NUM_DOCS = 512
+MULTIPLIER_NUM_DOCS = 4
+COUNT_NUM_DOCS = 10
+DEFAULT_BATCH_SIZE = 256
+COLUMN_OF_INTEREST = 'g:send' # to be changed to 'roundtrip' once issue is fixed
+
+
+class BenchmarkDriver(BaseDriver):
```

Comment:
Is it possible to define this in a separate file?

Reply:
Sure! Will do that